internal/operators/odf/validations.go

```diff
@@ -18,11 +18,17 @@ type odfClusterResourcesInfo struct {
 
 func (o *operator) validateRequirements(cluster *models.Cluster) (api.ValidationStatus, string) {
 	var status string
+	log := o.log
+
+	if cluster.ID != nil {
+		log = log.WithField("cluster", cluster.ID.String())
+	}
 
 	mode := getODFDeploymentMode(cluster, o.config.ODFNumMinimumHosts)
+	log.Debugf("ODF validate cluster - mode: %s", string(mode))
 
 	if mode == unknown {
-		status = "A cluster with only masters or with a minimum of 3 workers is required."
+		status = "The cluster must either have no dedicated worker nodes or at least three. Add or remove hosts, or change their roles configurations to meet the requirement."
```

Comment:
NIT:
Change to
"The cluster must either have no dedicated worker nodes or at least three. Add or remove hosts, or change their role configurations to meet the requirement."

Reply:
I agree with you, but I wanted to use the same as in - https://github.com/openshift/assisted-service/blob/0e3133ea2cef252e63573178db88186e84b927e1/internal/cluster/validator.go#L362 (I didn't wrote that one, AFAIU it is UX.) Lets keep compatibility